packages/backend/src/app/database/database-connection.ts

```diff
@@ -64,13 +65,14 @@ export const databaseConnection = new DataSource({
     username,
     password,
     database,
-    synchronize: false,
+    synchronize: true,
```

Comment:
Any reason to enable `synchronize`?

Reply:
No by mistake, thank you for catching this sorry